src/Utilities/ToolTask.cs

```diff
@@ -588,8 +588,20 @@ string responseFileSwitch
                 LogPrivate.LogWarningWithCodeFromResources("ToolTask.CommandTooLong", GetType().Name);
             }
 
+            // Do *not* set startInfo.CreateNoWindow = true.
+            // This causes the child process to be launched with its console code page (Win32 GetConsoleCP),
+            // and console output code page (Win32 GetConsoleOutputCP) reset to the OEM code page (Win32 GetOEMCP),
+            // rather than inherited from this process.
+            //
+            // This is problematic because processes normally write to their standard output using
+            // the console output code page. If that is reset to the default OEM code page,
+            // then it will be impossible to represent all Unicode characters,
+            // since the default configuration for all Windows OS languages is a non-Unicode-capable OEM code page.
+            //
+            // The only way to get Unicode output through the console is if an ancestor process
+            // set the console output code page to UTF-8 and text-producing programs honor it.
             ProcessStartInfo startInfo = new ProcessStartInfo(pathToTool, commandLine);
-            startInfo.CreateNoWindow = true;
+            startInfo.CreateNoWindow = false;
```

Comment:
Note: I do not know the ramifications of changing this outside of what I'm trying to fix, if any. The [documentation](https://docs.microsoft.com/en-us/windows/win32/procthread/process-creation-flags) says:

> CREATE_NO_WINDOW 0x08000000
> The process is a console application that is being run without a console window. Therefore, the console handle for the application is not set. This flag is ignored if the application is not a console application, or if it is used with either CREATE_NEW_CONSOLE or DETACHED_PROCESS.

Reply:
If this is problematic, I propose an alternate solution in my comment on #4870